README.md

```diff
@@ -34,14 +34,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+Semantic html is easier to read.
+
 2. Name two big differences between ```display: block;``` and ```display: inline;```.
 
+Inline doesnt respect height and width properties. Block elements break the flow of content on a new line. Inline allows elements to be stacked next to each other.
+
 3. What are the 4 areas of the box model?
 
+Content, Padding, Border, Margin.
+
 4. While using flexbox, what axis does the following property work on: ```align-items: center```?
 
+The Cross Axis
+
 5. Explain why git is valuable to a team of developers.
 
+It is easier to view everyones work in group projects through git.
+
```

Comment:
it also gives you save points that you can revert back to if a bug gets introduced and you need to go back to an earlier time in the code